register.html

```diff
@@ -1,47 +1,170 @@
 <!DOCTYPE html>
```

Comment:
This file is named `register.html` but the other two html files are pointing to `contact.html`. Either update those pages to point to `register.html` or name this file `contact.html`